Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.ts

```diff
@@ -14,10 +18,48 @@ import { SetDisplayedNotificationVersion } from 'src/app/shared/states';
 })
 export class DisplayNotificationDialog implements OnInit, OnDestroy {
   public static version: number = 1;
-  constructor(public dialogRef: MatDialogRef<DisplayNotificationDialog>, store: Store) {
+  constructor(
+    public dialogRef: MatDialogRef<DisplayNotificationDialog>,
+    store: Store,
+    private router: Router,
+    private snackBar: MatSnackBar,
+    private i18n: I18n,
+    private sanitizer: DomSanitizer
+  ) {
     store.dispatch(new SetDisplayedNotificationVersion(DisplayNotificationDialog.version));
   }
 
+  public openSnackBar() {
+    const url = this.sanitizer.bypassSecurityTrustHtml(
+      '<a href="' + window.location.href + '?skip_dialog=true">Link</a>'
+    );
+    const snackBarRef = this.snackBar
+      .open(
+        this.i18n({
+          meaning: 'Display-New-Url-Advice',
+          description: 'Message for new no cookie url',
+          id: 'PageInformationNewUrlNoCookies',
+          value:
+            'Add skip_dialog=true as parameter to url for skipping start dialogs if cookies are deleted after each session.'
+        }),
+        this.i18n({
+          meaning: 'Go-to-New-Url-Advice',
+          description: 'Message for following no cookie url',
+          id: 'PageInformationNewUrlNoCookiesUrl',
+          value: 'Go to Url'
+        }),
+        { duration: 6000 }
+      )
+      .onAction()
+      .subscribe(() => {
+        this.router.navigateByUrl('/?skip_dialog=true');
```

Comment:
because of the lines below, this line should be kept. Remove the two below

Reply:
`this.router.navigateByUrl('/?skip_dialog=true');` -->will set the router url, but page does not reload, so the user won't realy notice a change...i think this should be removed (this is useless at the moment, because site will reload in next line) 

`window.location.href = window.location.href + '?skip_dialog=true';` -->will reload the page with the parameter --> user notices the change, i think we should keep this